docs/general/electronics.md

```diff
@@ -87,16 +96,21 @@ There is no general standard for what colours are used for what voltage node, bu
 | 24V       | `24V`, `VCC`, `VMOT`, etc. | ![yellow wire]                |                                    |
 | 48V       | `48V`, `HT`, `VMOT`, etc.  | ![purple wire] ![orange wire] | Use a color different from 5V/24V  |
 
-## Chosing the correct wire type
+## Choosing the correct wire type
 
 {: .danger }
 > Using smaller wires than recommended for their rated current may result in cables melting or catching fire. Always use a wire gauge that at least matches the current load!
 
-Using appropriately sized wires ensures the cables can withstand the current draw on the wire. Using a too small wire gauge will increase the wire resistance and cause the cables to (over) heat. 
+Using appropriately sized wires ensures they can safely handle the current draw of your printer's components. Using undersized wires will increase the wire's electrical losses, which can lead to (over)heating, damage to the wires or your printer, or even other things. Undersized wiring is always an imminent fire hazard for this reason.
 
 ### Recommended wire gauges
 
-This table specifies recommended minimum gauges for nickle-free stranded copper wires; thicker wire can always be used. The rated ampacities are just a rule of thumb. At the rated limits the voltage drop (increases over distance), insulation temperature limit, thickness, thermal conductivity, and air convection and temperature should all be taken into account.
+This table specifies recommended minimum wire size when using nickle-free stranded copper wires; larger wire can always be used (within reasonable limits).
+
+The rated current capacities are a general rule of thumb. Ensure wires are never used beyond their rated specifications! This includes (but is not limited to): current rating, resistance per length, insulation temperature limit, and environmental temperature. The [Standard Wire Gauge] or [American Wire Gauge] engineering tables contain more reference information.
```

Comment:
Mentioned tables are for solid core, values in the table are for multicore. So this is may lead to confusion.

Reply:
I'd like to provide some citation for the "keep in mind these specifications" part, as it feels slightly unfinished without. Any suggestions on that, as alternatives to the links currently present?